src/Compute/v2/Models/QuotaSet.php

```diff
@@ -0,0 +1,170 @@
+<?php declare(strict_types=1);
+
+namespace OpenStack\Compute\v2\Models;
+
+use OpenStack\Common\Resource\Deletable;
+use OpenStack\Common\Resource\OperatorResource;
+use OpenStack\Common\Resource\Retrievable;
+use OpenStack\Common\Resource\Updateable;
+
+/**
+ * Represents a Compute v2 Quota Set
+ *
+ * @property \OpenStack\Compute\v2\Api $api
+ */
+class QuotaSet extends OperatorResource implements Retrievable, Updateable, Deletable
+{
+    /**
+     * The number of allowed instance cores for each tenant.
+     *
+     * @var  int|array
+     */
+    public $cores;
+
+    /**
+     * The number of allowed fixed IP addresses for each tenant.
+     * Must be equal to or greater than the number of allowed instances.
+     *
+     * @deprecated Since 2.35
+     * @var  int|object
+     */
+    public $fixedIps;
+
+    /**
+     * The number of allowed floating IP addresses for each tenant.
+     *
+     * @deprecated Since 2.35
```

Comment:
Ah I see. To clear up confusion, can we expand the comment to:

```
@deprecated Since Nova v2.35. This attribute will eventually move to Neutron, it is advised you do not use this.
```